src/Cache.ts

```diff
@@ -2,30 +2,30 @@ import { ICache } from "configcat-common";
 import { ProjectConfig } from "configcat-common/lib/ProjectConfig";
 
 export class LocalStorageCache implements ICache {
-    cache:  { [sdkKey: string] : ProjectConfig; } = {};
+    cache:  { [key: string] : ProjectConfig; } = {};
```

Comment:
We have some other 'key' params, could be 'cachekey'

Reply:
Discussed in private